docs/Operators.md

```diff
@@ -5680,40 +5704,40 @@ expect(node, inputs=[a, b, c], outputs=[y],
 
 
 <details>
-<summary>default_no_bias</summary>
+<summary>default_scalar_bias</summary>
 
 ```python
 node = onnx.helper.make_node(
     'Gemm',
     inputs=['a', 'b', 'c'],
     outputs=['y']
 )
-a = np.random.ranf([3, 5]).astype(np.float32)
-b = np.random.ranf([5, 4]).astype(np.float32)
-c = np.zeros([1, 4]).astype(np.float32)
+a = np.random.ranf([2, 3]).astype(np.float32)
+b = np.random.ranf([3, 4]).astype(np.float32)
+c = np.random.ranf(1).astype(np.float32)
```

Comment:
This is still not a scalar. 
```
>>> np.random.ranf(1).shape
(1,)
```
As you already have a reference implementation, could we do
```
c= 3.14
```
?

Reply:
Whoops! Let me fix that!

Reply:
Okay, I think I've fixed this for real now, sorry about that :)